### ISSUE TITLE
Popup window location not correct

### ISSUE THREAD

**Issue by wsy** [steam3d/MagicPods-Windows#396]
Device: AirPods Pro
Computer: i7 6700k
Bluetooth: Intel AX200
AppVersion: 2.0.59.0 x64

![Problem screenshot](https://github.com/steam3d/MagicPods-Windows/assets/2684991/00697625-bf35-4cd2-9c9d-fec4e81f0bd0)

My computer has multiple displays.

The main display: DisplayM, 3840*2160@150% scale
![Main display setting](https://github.com/steam3d/MagicPods-Windows/assets/2684991/15ee515c-6baf-4e71-88f2-7c60133ccc8c)

Another display: DisplayA, 1920*1080@100%scale
![image](https://github.com/steam3d/MagicPods-Windows/assets/2684991/906157df-755e-4797-a1b5-ef3a63969155)

App setting:
PopupSize: Medium
PopupLocation: BottomRight corner
![AppSetting](https://github.com/steam3d/MagicPods-Windows/assets/2684991/48870a13-169f-44f0-a2b1-e64db466d9f7)

Steps to reproduce this bug:
1. Open one program (such as Windows Terminal) and operate it on DisplayM.
2. Open AirPods Pro's battery case and watch where the popup window will appear.
3. MagicPods popup window should and it did appear at the bottom right corner of DisplayM.
4. Operate one program on DisplayA.
5. Open AirPods Pro's battery case and watch where the popup window will appear.
6. MagicPods popup window  should appear at the bottom right corner of main display. However, it appeared at wrong position.

My guess of what happened in step 6:
1. MagicPods got the effective resolution of **_`DisplayM`_**, which is 2560x1440.
2. MagicPods got the DPI scale of **_`current display of active window`_**, which is 100%.
3. MagicPods display popup window at wrong pixel (2560,1440, 100%) instead of correct pixel (2560, 1440, 150%).

I think the bug is caused by "'**_`DisplayM`_**' is not necessarily '**_`current display of active window`_**'". Further more, if you want to always display popup window on main display, '**_`current display of active window`_**' is not necessarily where MagicPods' popup window will be displayed.

**Comment by steam3d** [comment]
Hi. Does the popup position right when you restart the app?

The MagicPods use primary monitor to show popup and subscribes to DPI changed event. I draw popup on this monitor (primary):

```
HMONITOR GetPrimaryMonitorHandle()
{
 const POINT ptZero = { 0, 0 };
 return MonitorFromPoint(ptZero, MONITOR_DEFAULTTOPRIMARY);
}
```

**Comment by steam3d** [comment]
Yep. I reproduced the issue, no idea why the window draws on secondary screen instead of main screen. I will check

**Comment by steam3d** [comment]
Fixed in 2.0.66.0
